### PR TITLE
Introducing test constants into separate files

### LINKED ISSUE #1271: Separate out all constants to its dedicated package to reuse
currently some of the tests are using local string constant, it should be carried out to dedicated `/tests/constants` package.

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
fixes #1271 

Testing done: Yes locally

```
PASS
2017/05/17 04:17:53 -----Clean-up finished - current time:  2017-05-17 04:17:53.127831766 +0000 UTC
ok  	github.com/vmware/docker-volume-vsphere/tests/e2e	125.418s
```

### CODE REVIEW

File: tests/constants/dockercli/cmd.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const (
+	docker    = "docker "
+	dockerVol = docker + "volume "
+
+	// ListVolumes to list down docker volumes
+	ListVolumes = dockerVol + "ls "
+
+	// Inspect to grab volume properties
+	Inspect = dockerVol + "inspect "
```
Comment (shaominchen) [inline tests/constants/dockercli/cmd.go]: Inspect => InspectVolume

File: tests/constants/dockercli/cmd.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+	// Remove constant refers delete volume command
+	Remove = dockerVol + "rm "
```
Comment (shaominchen) [inline tests/constants/dockercli/cmd.go]: Remove => RemoveVolume

File: tests/constants/dockercli/cmd.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A home to hold test constants related with docker cli.
+
+package dockercli
+
+const (
+	docker    = "docker "
```
Comment (shaominchen) [inline tests/constants/dockercli/cmd.go]: Nit: why these 2 names start with small case?

Comment (shuklanirdesh82) [inline tests/constants/dockercli/cmd.go]: kept it private members ... no need to export them

Comment (shaominchen) [inline tests/constants/dockercli/cmd.go]: Then above VmdkopsAdmin and VmdkopsAdminVolume should not be exported either.

File: tests/constants/dockercli/cmd.go (hunk 2 above)
Comment (shuklanirdesh82) [inline tests/constants/dockercli/cmd.go]: Done!

File: tests/constants/dockercli/cmd.go (hunk 1 above)
Comment (shuklanirdesh82) [inline tests/constants/dockercli/cmd.go]: Done!

File: tests/constants/dockercli/cmd.go (hunk 3 above)
Comment (shuklanirdesh82) [inline tests/constants/dockercli/cmd.go]: Agree, updating into the next diff.

**Pull request merged**